windows/hax_entry.h

```diff
@@ -163,4 +163,8 @@ extern PDRIVER_OBJECT HaxDriverObject;
 #define HAX_VM_IOCTL_NOTIFY_QEMU_VERSION \
         CTL_CODE(HAX_DEVICE_TYPE, 0x910, METHOD_BUFFERED, FILE_ANY_ACCESS)
 
+/* API version 5.0 */
+#define HAX_IOCTL_VCPU_DEBUG \
+        CTL_CODE(HAX_DEVICE_TYPE, 0x912, METHOD_BUFFERED, FILE_ANY_ACCESS)
```

Comment:
nit: I skipped `0x912` intentionally when I added the `ADD_RAMBLOCK` ioctl, because `0x910` appears twice... So I'd use `0x916` for this one, but that's not a big deal.